### PR TITLE
[release-4.13] WINC-1025: Enable CSI plugins on Windows nodes 

### PR DESCRIPTION
**Pull request by wgahnagl**
manual backport of https://github.com/openshift/windows-machine-config-operator/pull/1530

[[submodule] Add csi-proxy](https://github.com/openshift/windows-machine-config-operator/commit/a846321905d81d6d03b69e4ba8cbb0ebab40810d) 
[[services] Add csi-proxy to Windows service CM](https://github.com/openshift/windows-machine-config-operator/commit/41a3224c7107b832f44015f725049b96f8ba03d5)
[[test] Add vSphere storage testing](https://github.com/openshift/windows-machine-config-operator/commit/c051f8ec1047a5be65df0cd7bb0fde1db3fc70c0)

### CODE REVIEW

**Review by sebsoto — APPROVED** [review]
Good work, left a comment, you probably can just remove all changes with the test suite. But you should rerun the e2e tests locally and see if that ends up working as expected.

File: test/e2e/providers/vsphere/vsphere.go [hunk 1]
```diff
@@ -6,9 +6,11 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"reflect"
```
[the remark this answers is not in the document]
Comment (wgahnagl) [reply test/e2e/providers/vsphere/vsphere.go:9]: do you mean tremove all of the e2e tests for the 4.13 version?

**Review by jrvaldes — CHANGES_REQUESTED** [review]
@wgahnagl  Seems this is a manual backport for:
- https://github.com/openshift/windows-machine-config-operator/pull/1530

Please reflect that in the description of the PR. For example: "Manual backport of https://github.com/openshift/windows-machine-config-operator/pull/1530"


In addition, consider adding the `-x` flag while cherry-picking the commits so that the original hash is included in the footer of the commit message.

File: test/e2e/providers/vsphere/vsphere.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (sebsoto) [reply test/e2e/providers/vsphere/vsphere.go:9]: Not all of the e2e tests. The test changes introduced in this PR.

[the remark this answers is not in the document]
Comment (wgahnagl) [reply test/e2e/providers/vsphere/vsphere.go:9]: sorry, I mean should I drop [this commit](737bec68e4ed7fc96aa8657b33fa3baa1686fce6) ?

**Comment by jrvaldes** [comment]
/close 

in favor of https://github.com/openshift/windows-machine-config-operator/pull/1721

**Pull request closed without merging**